src/components/Book.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Books from './Books';
+import './Book.css';
```

Comment:
- [ ] Kindly when I tried to remove it, the project runs but there are issues with the styling, Kindly fix them or add the `Book.css` file.

![image](https://user-images.githubusercontent.com/35535130/174116648-25a935a5-e831-4a0d-bef1-98fab792a0dd.png)